src/hooks/useStore.js

```diff
@@ -6,6 +6,9 @@ const useStore = create(set => ({
 		{id: nanoid(), name: 'Abwaschen', time: 15},
 		{id: nanoid(), name: 'Post sortieren', time: 2},
 	],
+	addTask(name, time) {
+		set(({tasks}) => ({tasks: [{id: nanoid(), name, time}, ...tasks]}));
+	},
 
 	//Beispielcode
```

Comment:
Please delete all unused code.

Reply:
I can delete the unused code in this branch? I forgot it at the beginning and thought I need an extra user story for that.